scripts/test/watch

```diff
@@ -1,12 +0,0 @@
-#!/bin/sh
-
-set -e
-
-filewatcher \
-    -L 5 \
-    -x '**/*.swp' \
-    -x .git \
-    -x build \
-    -x .idea \
-    -- \
-    sh -c 'go test -timeout 10s -v ./${dir} || ( echo; echo; exit 1 )'
```

Comment:
@dnephin It is a dependency that needs to downloaded and a workflow that doesn't work very well in Go. Including this creates cognitive burden in that people may think they need to use it. Test on change is a workflow item and should be removed.